compiler/rustc_borrowck/src/session_diagnostics.rs

```diff
@@ -157,3 +157,23 @@ pub(crate) enum RequireStaticErr {
         multi_span: MultiSpan,
     },
 }
+
+#[derive(SessionSubdiagnostic)]
+pub(crate) enum RegionNameLables {
+    #[label(borrowck::lifetime_defined_here)]
+    LifetimeDefinedHere {
+        rg_name: String,
+        #[primary_span]
+        span: Span,
+    },
+}
+pub(crate) enum RequireBorrowLasts {
```

Comment:
I think you need to introduce lifetime `'a`.

```suggestion
pub(crate) enum RequireBorrowLasts<'a> {
```